cpp/velox/operators/writer/VeloxParquetDatasource.h

```diff
@@ -69,6 +69,19 @@ class VeloxParquetDatasource final : public Datasource {
     return schema_;
   }
 
+  bool isSupportedS3SdkPath(const std::string& filePath_) {
+    // support scheme
+    const std::array<const char*, 4> supported_schemes = {"s3a:", "oss:", "cos:", "cosn:"};
```

Comment:
do we support s3:// ? I remember we can support with a config, @yma11 , this one?
.set('spark.hadoop.fs.s3.impl','org.apache.hadoop.fs.s3a.S3AFileSystem')

Reply:
I remember it doesn't work if the path started with `s3://`, we don't have changed the conf as you mentioned. As mentioned [here](https://hadoop.apache.org/docs/r3.1.1/hadoop-aws/tools/hadoop-aws/index.html#Other_S3_Connectors), `s3a` should be the one we support.

Reply:
Just tried, with the config, we can read s3:// files. Looks it's a spark feature

Reply:
thanks all, `s3` scheme added